src/lib.rs

```diff
@@ -62,7 +62,11 @@ pub mod arm;
 #[cfg(any(target_arch = "aarch64", target_arch = "arm64ec"))]
 pub mod aarch64;
 
-#[cfg(all(target_arch = "aarch64", target_os = "linux", not(feature = "no-asm"),))]
+#[cfg(all(
+    any(target_arch = "aarch64", target_arch = "arm64ec"),
```

Comment:
ARM64EC only exists on Windows.

Reply:
Removed this

Reply:
Actually, would it be better to leave arm64ec but change the cfg to `not(target_os = "macos")` so it still gets tested on Windows? I don't know where this is needed but it must have been passing okay on the windows platforms.